src/components/ChartCard.js

```diff
@@ -48,10 +48,18 @@ export class ChartCard extends React.Component {
         <div className="experiment-count-container">
          {this.props.experimentCount ?( <span className="experiment-count">
             {this.props.experimentCount} Chaos Experiments
-          </span>):(<span></span>)}
+          </span>):
+          (<span className="experiment-chaos-type" >
+             {this.props.chaosType ?( <i className ="mi-application infra-icon">
+             <span className="chaos-tooltiptext ">
+                    { this.props.chartType=="generic" ? ('Infra-Chaos :- Multiple application might be impacted')
```

Comment:
multiple "applications" might be impacted

Reply:
if 
    this.props.chaosType   == > true
else
  <span></span>

It will impact only, mentioned chaosType in experiment chart. otherwise will be blank